build.gradle.kts

```diff
@@ -29,20 +29,20 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-actuator")
     implementation("org.springframework.boot:spring-boot-starter-jdbc")
     implementation("org.apache.commons:commons-lang3")
-    implementation("com.google.guava:guava:28.1-jre")
+    implementation("com.google.guava:guava:29.0-jre")
     implementation("com.github.ben-manes.caffeine:caffeine")
-    implementation("com.oracle:ojdbc8:12.2.0.1")
+    implementation("com.oracle:ojdbc8:18.3")
```

Comment:
Er dette testet? Ikke sikkert at nyere oracle drivere virker likt mot dagens servere. #

Reply:
Hvordan går jeg frem for å teste dette? Evt bør vi la vær å oppgradere denne og heller ta den i en høstrengjørring?

Reply:
Hvis testene passerer på Jenkins bør alt være greit :+1: